### ISSUE TITLE
Missing toolbox.json

### ISSUE THREAD

**Issue by fennectech** [plutooo/nx-btred#20]
Your missing a toolbox.json file for management via variouis toolbox apps.

**Comment by Slluxx** [comment]
Agreed

**Comment by UnderPrivlidged** [comment]
I mean, anyone can easily make one themselves.

**Comment by fennectech** [comment]
Yes.  But there is no reason not to include this in the zip lol.  It less than a single cluster.  The average user isnt going to be able to use it after install as it doesnt appear in any of the sysmodule managers.
